libs/add_compat_headers.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+# Script to introduce the deprecated include after factorazing in modules
+# This file is globbing through the include_compatibility folder, so basic 
+# files should already be here
+
+script_source=0
```

Comment:
```suggestion
script_sourced=0
```
?
Does that mean this can be either run or sourced?

Reply:
I was running either sourced or executed but the problem was to exit the script if you had no arguments specified, a return in a unsource script would just exit my ssh session ^^